ruff.toml

```diff
@@ -0,0 +1,44 @@
+target-version = "py39"
+
+line-length = 88
+
+extend-exclude = ["examples"]
```

Comment:
I have excluded the examples notebooks, happy to add this later on is this would be helpful?